.travis.yml

```diff
@@ -10,11 +10,7 @@ git:
 
 addons:
   apt:
-    sources:
-      - ubuntu-toolchain-r-test
-      - sourceline: 'ppa:kzemek/boost'
```

Comment:
Ubuntu 16.04 apt sources already have all the dependencies we need

Reply:
I'd add here the update instruction

```
addons:
  apt:
    update: true
```